src/PrestaShopBundle/Form/Admin/Catalog/Category/AbstractCategoryType.php

```diff
@@ -122,19 +155,45 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
             ])
             ->add('active', SwitchType::class, [
+                'label' => $this->trans('Displayed', 'Admin.Global'),
+                /** IMO help here is not exactly clear. I don't understand what click on "displayed" means, maybe previously it was clickable but now isin't? */
+                'help' => $this->trans(
+                        'Click on "%displayed_label%" to index the category on your shop.',
+                        'Admin.Catalog.Help',
+                        [
+                            '%displayed_label%' => $this->trans('Displayed', 'Admin.Global')
+                        ]
+                    ) . '<br>' .
+                    $this->trans(
```

Comment:
I wonder if we should create a Helper class to handle these. This
```
'[1]' => '<a href="' . $this->router->generate('admin_module_manage') .  '" target="_blank" rel="noopener noreferrer nofollow">',
 '[/1]' => '</a>
 ```
 looks weird.
 
 Something like `HelperTrans::wrapTransIntoATag()` ?

Reply:
I don't see how you would do this without separating 
`"If you want a category to appear in the menu of your shop, go to [1]Modules > Module Manager[/1] and configure your menu module."` into 2 separate strings. 

And if you do separate it into 2 strings then I don't think helper is needed as you can just:
`$string = '<a href="' . $this->router->generate('admin_module_manage') . '" target="_blank" rel="noopener noreferrer nofollow">' . $this->trans('Modules > Module Manager', 'Dom') . '</a>';`
And pass that variable to the translation.

I agree it looks weird, but I don't think there is a better way to add HTML tags and keep it as one translatable string.